styles.css

```diff
@@ -0,0 +1,231 @@
+  background: #fff;
+  width: 95%;
+}
+
+nav .navbar-logo {
+  color: #6070ff;
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+nav .navbar-hamburger {
+  display: flex;
+  flex-direction: column;
+  height: min-content;
+  margin: 18px -10px 0 0;
+  cursor: pointer;
+}
+
+nav .navbar-hamburger .bar {
+  width: 20px;
+  height: 3px;
+  margin-bottom: 4px;
+  background: #6070ff;
+  border-radius: 18%;
+  transition: 0.4s;
+  position: relative;
+  top: 21px;
+}
+
+section {
+  display: flex;
+  justify-content: space-between;
+  border-bottom-left-radius: 50%;
+}
+
+#ellipse1 {
+  position: absolute;
+  background-image: url('images/ellipse1.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  top: 18%;
+  left: -3%;
+  width: 45px;
+  height: 25px;
+}
+
+#rentangle {
+  position: absolute;
+  background-image: url('images/rectangle.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 40px;
+  height: 20px;
+  top: 20%;
+  left: 30%;
+}
+
+#ellipse5 {
+  position: absolute;
+  background-image: url('images/circle1.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 15px;
+  height: 14px;
+  top: 21%;
+  left: 30.8%;
+}
+
+#rectangle2 {
+  position: absolute;
+  background-image: url('images/rectangle2.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 122px;
+  height: 50px;
+  top: 16%;
+  right: -7.3%;
+}
+
+#ellipse3 {
+  position: absolute;
+  background-image: url('images/ellipse3.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 122px;
+  height: 80px;
+  top: 20%;
+  right: -10%;
+}
+
+#circle1 {
+  position: absolute;
+  background-image: url('images/circle1.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  width: 12px;
+  height: 12px;
+  top: 44%;
+  left: 70%;
+}
+
+#ellipse2 {
+  position: absolute;
+  background-image: url('images/ellipse2.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  top: 72.5%;
+  left: 75%;
+  width: 25px;
+  height: 25px;
+}
+
+#ellipse4 {
+  position: absolute;
+  background-image: url('images/ellipse4.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  top: 75%;
+  left: 73%;
+  width: 25px;
+  height: 25px;
+}
+
+#circle2 {
+  position: absolute;
+  background-image: url('images/circle2.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  top: 88%;
+  left: 28%;
+  width: 16px;
+  height: 16px;
+}
+
+#circle3 {
+  position: absolute;
+  background-image: url('images/circle3.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  top: 88%;
+  left: 63%;
+  width: 20px;
+  height: 20px;
+}
+
+#rectangle3 {
+  position: absolute;
+  background-image: url('images/rectangle3.svg');
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  top: 88.3%;
+  left: 40.5%;
+  width: 140px;
+  height: 30px;
+}
```

Comment:
Well done so far 💯 as requested by the previous reviewer [here](https://github.com/migcm06/miguel-portfolio/pull/2#discussion_r1180845127) make sure that the section has the proper background image (decorations are always background images, not `img` tags).

If an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.